### PR TITLE
new tinderbox prototype in Python

### PR DESCRIPTION
**Pull request by monsieurp**
This is a first stab at fixing #2.

### CODE REVIEW

File: tb/commands.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import subprocess
+import shlex
+
+
+def run(cmd, env=None):
+    if cmd == None or cmd == "":
```
Comment (alip) [inline tb/commands.py]:
```suggestion
    if not cmd:
```

File: tb/emerge.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+--update \
+--with-bdeps=y \
+""".strip()
+
+
+def eformat(bin_exclude=yml["BINEXCLUDE"],
+        exclude=yml["EXCLUDE"],
+        target="/",
+        jobs=yml["JOBS"],
+        root=yml["ROOTSDIR"],
```
Comment (alip) [inline tb/emerge.py]: `root` is unused now that we use `target` to set the root directory to the emerge command below.

**Review by alip — COMMENTED** [review]
lgtm in general, noted a few small improvements.
Let's also add a `.gitignore` file to ignore byte-compiled Python modules, ie `*.py[co]`.

File: tb/commands.py (hunk 1 above)
Comment (monsieurp) [reply tb/commands.py]: fixed thanks :)

File: tb/emerge.py (hunk 2 above)
Comment (monsieurp) [reply tb/emerge.py]: fixed!

**Comment by monsieurp** [comment]
@alip .gitignore file added to the repo, thanks!

**Review by alip — APPROVED** [review]

**Pull request merged**